packages/web3/std/nft_collection_interface.ral

```diff
@@ -1,8 +1,5 @@
 Interface INFTCollection {
-   event Minted(minter: Address, tokenIndex: U256, tokenId: ByteVec)
-
-   pub fn getName() -> ByteVec
-   pub fn getSymbol() -> ByteVec
```

Comment:
Maybe we should keep these two most important fields on-chain so that they are totally immutable?

Reply:
We decided not to having these two following EIP1155.